dist/unix/CMakeLists.txt

```diff
@@ -18,17 +18,20 @@ if (SYSTEMD)
     )
 endif()
 
-if (GUI)
-    list(APPEND MAN_FILES ${PROJECT_SOURCE_DIR}/doc/qbittorrent.1)
-else()
-    list(APPEND MAN_FILES ${PROJECT_SOURCE_DIR}/doc/qbittorrent-nox.1)
-endif()
-
-install(FILES ${MAN_FILES}
-    DESTINATION ${CMAKE_INSTALL_MANDIR}/man1
-    COMPONENT doc
+set(MAN_PAGE_LANGUAGES
```

Comment:
Couldn't they be detected automatically?

Reply:
That's possible with a regex, yes. I would be willing to do that.

Reply:
Alright, changed my mind on using a regex because I can't figure out which language code format we use and which format man expects. The script now assumes that any subdirectory of `doc/` is a translation.

Reply:
> Couldn't they be detected automatically?

However, I doubt whether the game is worth the candle.

Reply:
Ultimately your call, I don't feel strongly either way.

Reply:
Reverted back to the manual list. Apologies for the delay.